docs/usage/self-hosted-configuration.md

```diff
@@ -11,6 +11,11 @@ Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md
 
 ## allowCustomCrateRegistries
 
+## allowGitLfs
+
+**Note** You need to also set the Git commands that Renovate passes to the `gitNoVerify` option.
+This way Renovate can commit and push Git LFS objects.
```

Comment:
```suggestion
Use this option to allow Renovate to work with repositories which use Git LFS (Git Large File Support).

**Note** You need to also set the Git commands that Renovate passes to the `gitNoVerify` option.
This way Renovate can commit and push Git LFS objects.
```

How about a short introductory sentence, that explains what this option does?

Or do we think the section name is good enough on its own?

Reply:
Remember, that the docs compiler will also prepend the message and default values from config definitions file.